clang/test/SemaTemplate/temp_arg_nontype_cxx20.cpp

```diff
@@ -62,7 +62,7 @@ namespace ClassNTTP {
   template<A a> constexpr int f() { return a.y; }
   static_assert(f<A{1,2}>() == 2);
 
-  template<A a> int id;
+  template<A a> int id; // expected-note {{passing argument to parameter 'a' here}}
```

Comment:
What diagnostic is this note associated with?

Reply:
L70. you want me to use a label i guess?

Reply:
Ahhh, it was just out of sight. Yeah might as well switch to a bookmark instead. I don't feel that strongly though.